dotCMS/src/main/java/com/dotmarketing/business/VersionableFactoryImpl.java

```diff
@@ -192,25 +216,19 @@ protected VersionInfo getVersionInfo(String identifier) throws DotDataException,
             	vi.setIdentifier(identifier);
             	vi.setWorkingInode("NOTFOUND");
             }
-            icache.addVersionInfoToCache(vi);
+            this.icache.addVersionInfoToCache(vi);
         }
-        if(vi.getWorkingInode().equals("NOTFOUND"))
+        if(vi.getWorkingInode().equals("NOTFOUND")) {
```

Comment:
The method containing that code is not part of the issue. I can update it though.